content/kubermatic/master/data/seed.yaml

```diff
@@ -131,23 +129,6 @@ spec:
           # 'Default' or 'None'. Defaults to "ClusterFirst". DNS parameters given in DNSConfig will be merged with the
           # policy selected with DNSPolicy.
           dns_policy: ""
-        # Nutanix is experimental and unsupported
-        nutanix:
-          # Optional: AllowInsecure allows to disable the TLS certificate check against the endpoint (defaults to false)
-          allow_insecure: false
-          # Endpoint to use for accessing Nutanix Prism Central. No protocol or port should be passed,
-          # for example "nutanix.example.com" or "10.0.0.1"
-          endpoint: ""
-          # Images to use for each supported operating system
-          images:
-            amzn2: ""
-            centos: ""
-            flatcar: ""
-            rhel: ""
-            sles: ""
-            ubuntu: ""
-          # Optional: Port to use when connecting to the Nutanix Prism Central endpoint (defaults to 9440)
-          port: 9440
```

Comment:
This doesn't look right, why is it getting reverted? This was added as part of https://github.com/kubermatic/docs/commit/afaf873ed279d9b404495691a1bd0300abc0b0d7 syncing from KKP `master´.

Reply:
This was generated by `./hack/ci/update-docs.sh` and I blindly trusted the results. I will drop this commit. Thanks!

Reply:
Thanks for checking again!